### ISSUE TITLE
Screen Reader: The captions button should announce it’s state

### ISSUE THREAD

**Issue by miesgre** [polimediaupv/paella-basic-plugins#6]
The captions button should announce it’s state. It’s not saying if closed captions are on or off.

@karendolan

**Comment by ferserc1** [comment]
I have a question about this issue

As the captions are implemented in paella-core, it is not clear to me that they are sufficiently readable for a person who needs a screen reader to activate the captions.

Do you have any user feedback on this issue? I am not saying that this issue should not be prioritized, but rather that if the subtitles are not readable, then perhaps that part should be prioritized rather than indicating the status of the subtitles on the screen reader.
